xml/depl_require.xml

```diff
@@ -2069,6 +2069,24 @@
         between the cluster nodes) to ensure redundancy.
        </para>
       </important>
+      <important>
+       <title>Dedicated networks</title>
+       <para>
+        The <literal>corosync</literal> network communication layer is
```

Comment:
Hmm not quite - that's too prescriptive.  The OpenStack traffic could be spread over multiple interfaces, some or all of which are bonded.